src/components/Message/MessageSimple/MessageFooter.tsx

```diff
@@ -154,3 +170,117 @@ export const MessageFooter = <
     </View>
   );
 };
+
+const areEqual = <
+  At extends UnknownType = DefaultAttachmentType,
+  Ch extends UnknownType = DefaultChannelType,
+  Co extends string = DefaultCommandType,
+  Ev extends UnknownType = DefaultEventType,
+  Me extends UnknownType = DefaultMessageType,
+  Re extends UnknownType = DefaultReactionType,
+  Us extends UnknownType = DefaultUserType
+>(
+  prevProps: MessageFooterPropsWithContext<At, Ch, Co, Ev, Me, Re, Us>,
+  nextProps: MessageFooterPropsWithContext<At, Ch, Co, Ev, Me, Re, Us>,
+) => {
+  const {
+    alignment: prevAlignment,
+    formattedDate: prevFormattedDate,
+    members: prevMembers,
+    message: prevMessage,
+    otherAttachments: prevOtherAttachments,
+    showMessageStatus: prevShowMessageStatus,
+  } = prevProps;
+  const {
+    alignment: nextAlignment,
+    formattedDate: nextFormattedDate,
+    members: nextMembers,
+    message: nextMessage,
+    otherAttachments: nextOtherAttachments,
+    showMessageStatus: nextShowMessageStatus,
+  } = nextProps;
+
+  const alignmentEqual = prevAlignment === nextAlignment;
+  if (!alignmentEqual) return false;
+
+  const membersEqual = prevMembers === nextMembers;
```

Comment:
members is an object, so better to check with caution - similar to how its done in `MessageContent`
https://github.com/GetStream/stream-chat-react-native/blob/master/src/components/Message/MessageSimple/MessageContent.tsx#L495

![Screenshot 2021-04-28 at 20 45 44](https://user-images.githubusercontent.com/11586388/116456727-dba1af80-a862-11eb-8cd9-2ea81f35f95f.png)